src/pages/guides/quickstart/index.md

```diff
@@ -140,7 +155,8 @@ Here are the minimum browser requirements for the SDK:
 | Microsoft Edge | 107 and onwards
 | Firefox | 117 and onwards
 
-Notes:
+**Notes:**
```

Comment:
As an additional information block.

Reply:
Also, was the browser support tested and verified?

Reply:
Next Steps can also point them to the V4 Sample file in github. We can add a discover block there.

Reply:
I added the info block. I've asked if the browser support still holds true.
A few days ago, I asked Ruben whether we should have a separate repo for samples (I think it would be appropriate, also for the new ones I'm working on).
I agree on linking the V4 Sample, let's remember to add the first couple of tutorials there as well, when they're ready.